file-formats/dmon/dmon-v1.json

```diff
@@ -111,10 +111,31 @@
         "System Daemon"
       ],
       "enumDescriptions": [
-        "Application daemon",
-        "System daemon"
+        "Application Daemon",
+        "System Daemon"
```

Comment:
Is titlecase intended for "Daemon" in descriptions?

Reply:
Daemon is an official SAP Term, I think that was the reason why I wrote it in upper case